tsconfig.json

```diff
@@ -6,6 +6,9 @@
         "experimentalDecorators": true,
         "moduleResolution": "node",
         "sourceMap": true,
+        "declaration": true,
+        "declarationMap": true,
+        "declarationDir": "build",
```

Comment:
is this a problem since build isn't committed or will the map also be moved to releases?